### ISSUE TITLE
Unable to find GlideModule implementation

### ISSUE THREAD

**Issue by etam** [schaal/ocreader#86]
Version: 0.40 installed from f-droid
OS: LineageOS 14.1

The application fails to start with the following error:

```
ACRA caught a RuntimeException for email.schaal.ocreader
java.lang.RuntimeException: Unable to start activity ComponentInfo{email.schaal.ocreader/email.schaal.ocreader.ListActivity}: java.lang.IllegalArgumentException: Unable to find GlideModule implementation
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2684)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2751)
	at android.app.ActivityThread.-wrap12(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1496)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6186)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
Caused by: java.lang.IllegalArgumentException: Unable to find GlideModule implementation
	at com.bumptech.glide.module.ManifestParser.parseModule(ManifestParser.java:69)
	at com.bumptech.glide.module.ManifestParser.parse(ManifestParser.java:47)
	at com.bumptech.glide.Glide.initializeGlide(Glide.java:228)
	at com.bumptech.glide.Glide.initializeGlide(Glide.java:219)
	at com.bumptech.glide.Glide.checkAndInitializeGlide(Glide.java:180)
	at com.bumptech.glide.Glide.get(Glide.java:164)
	at com.bumptech.glide.Glide.getRetriever(Glide.java:670)
	at com.bumptech.glide.Glide.with(Glide.java:697)
	at email.schaal.ocreader.OCReaderBaseApplication$1.cancel(OCReaderBaseApplication.java:75)
	at com.mikepenz.materialdrawer.util.DrawerImageLoader.cancelImage(DrawerImageLoader.java:65)
	at com.mikepenz.materialdrawer.AccountHeaderBuilder.setImageOrPlaceholder(AccountHeaderBuilder.java:1202)
	at com.mikepenz.materialdrawer.AccountHeaderBuilder.buildProfiles(AccountHeaderBuilder.java:1088)
	at com.mikepenz.materialdrawer.AccountHeaderBuilder.build(AccountHeaderBuilder.java:865)
	at email.schaal.ocreader.ListActivity.onCreate(ListActivity.java:242)
	at android.app.Activity.performCreate(Activity.java:6684)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2637)
	... 9 more
Caused by: java.lang.ClassNotFoundException: com.bumptech.glide.integration.okhttp3.OkHttpGlideModule
	at java.lang.Class.classForName(Native Method)
	at java.lang.Class.forName(Class.java:400)
	at java.lang.Class.forName(Class.java:326)
	at com.bumptech.glide.module.ManifestParser.parseModule(ManifestParser.java:67)
	... 25 more
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.bumptech.glide.integration.okhttp3.OkHttpGlideModule" on path: DexPathList[[zip file "/data/app/email.schaal.ocreader-1/base.apk"],nativeLibraryDirectories=[/data/app/email.schaal.ocreader-1/lib/arm, /data/app/email.schaal.ocreader-1/base.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
	... 29 more
```

**Comment by schaal** [comment]
Just released version 0.41 with a quick fix for this crash, should be on f-droid in a couple of days

**Comment by schaal** [comment]
New fixed version is now available on F-Droid
